### ISSUE TITLE
Add support for selecting the initial file

### ISSUE THREAD

**Issue by ptzz** [gokcehan/lf#101]
Hi, I've created a [vim plugin](https://github.com/ptzz/lf.vim) that supports both `vim` and `neovim`. One thing that would be nice to have is ability to have `lf` highlight an initial file on startup, i.e. something corresponding to `--selectfile` option of `ranger`.

**Comment by gokcehan** [comment]
Sorry, I forgot to reply to this.

Currently when a directory is passed to `lf` it starts in that directory. When I was implementing it, I was thinking when the argument is a file it should select the file instead. However, this turned out to be a little difficult since you need to instantiate the navigation and read the directory before you can select a file. This is not the case for directories since you can simply chdir to the directory when you are reading the arguments before starting the application. I will try to see if there is an easy way to do this when I have some time without messing things up too much.

For the multiple file case, `select` command currently takes only a single argument. Maybe we can change it in such a way that when a single argument is given, it should select that file, when multiple arguments are given, it should mark those files. We could then simply consider this feature as passing arguments to a `select` command.

**Comment by gokcehan** [comment]
I have now added initial file selection (i.e. `lf foo.txt` will select `foo.txt` on startup). This is implemented simply by running a `select` command on startup so initial directory also still works (i.e. `lf foo` will change directory to `foo` on startup). 

@TeddyDD I mentioned we could change `select` command to accept multiple arguments but I'm not so sure about it anymore. For regular expression selection, I think we may add different commands. And then these can be used with the new `-command` flag that you implemented.

**Comment by TeddyDD** [comment]
yeah, select seems to make sense with single argument.

@ptzz to the original issue: As @gokcehan said you will be able to `lf -select 'file' ~/Dir` in next release of lf. 

You can also do as I did in Kakoune integration plugin - obtain client id and send commands from Vim after lf is launched.

**Comment by gokcehan** [comment]
@TeddyDD I have not added a `-select` switch for this. You just need to pass the filename as the argument. If I understand the intention of that example, it should be `lf ~/Dir/file`

**Comment by TeddyDD** [comment]
my bad, I meant `lf -command 'select file' dir`

**Comment by gokcehan** [comment]
I was thinking multiple selection with `select` should have been working with something like:

    lf -command 'select foo.txt ; toggle ; select bar.txt ; toggle'

But it seems that `toggle` command does not work at startup before directory initialization and honestly I don't quite understand how `select` works at startup. We could try to add a new command for multiple selection with a glob pattern but I doubt that command would work at startup either since it requires toggling.

Anyway, initial file selection is already implemented, so this issue is getting old. I'm closing this issue now. Feel free to open a new issue for glob selection if anyone wants.
